index.html

```diff
@@ -44,14 +44,16 @@
           // only "name" is required
           editors: [{
             name: "Manu Sporny", url: "https://digitalbazaar.com/",
-            company: "Digital Bazaar", companyURL: "http://digitalbazaar.com/"
+            company: "Digital Bazaar", companyURL: "http://digitalbazaar.com/",
+            w3cid: 41758
           }, {
             name: "Dave Longley", url: "https://digitalbazaar.com/",
             company: "Digital Bazaar", companyURL: "http://digitalbazaar.com/",
-            note: "2014-2022"
+            note: "2014-2022", w3cid: 48025
           }, {
             name: "Mike Prorock", url: "https://www.linkedin.com/in/mprorock",
-            company: "Mesur.io", companyURL: "https://mesur.io/"
+            company: "Mesur.io", companyURL: "https://mesur.io/",
+            w3cid: 0
```

Comment:
@mprorock, we'll need your w3cid, which you can find from this page: https://www.w3.org/users/myprofile

Reply:
It is 130636, @msporny @mprorock

Reply:
Fixed in bf73e97442a4fd82d580587db897ae427234e0be.